configure.ac

```diff
@@ -107,5 +107,20 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <rdma/fabric.h>
      AC_MSG_WARN([usnic-tools requires libfabric >=v1.3])
      AC_MSG_ERROR([Cannot continue])])
 
+# See if RPATH support works
+LDFLAGS_save=$LDFLAGS
+LDFLAFGS="$LDFLAGS_save -Wl,-rpath -Wl,$fab_libdir"
+AC_CHECK_LIB([fabric], fi_version,
+	     [rpath_works=1],
+	     [rpath_works=0
+	      LDFLAGS=$LDFLAGS_save])
+
+# If RPATH works, see if RUNPATH works, too
```

Comment:
This comment seems to suggest that this could fail, what happens if it fails? Are you supposed to be resetting `$LDFLAGS_save`?